lib/cli.js

```diff
@@ -16,13 +16,13 @@ const Router = require('./router');
 
 const gens = list(process.argv.slice(2));
 
-// Override http networking to go through a proxy ifone is configured
+// Override http networking to go through a proxy if one is configured
 require('global-tunnel-ng').initialize();
 
-/* eslint new-cap: 0, no-extra-parens: 0 */
-const tabtab = new Tabtab.Commands.default({
+const tabtab = new Tabtab.Commands({
   name: 'yo',
-  completer: 'yo-complete'
+  completer: 'yo-complete',
+  cache: !process.env.YO_TEST
```

Comment:
The cache was a nightmare to test and to prone for errors, I decided to remove this feature in the upcoming tabtab version